radio/src/gui/gui_common.h

```diff
@@ -123,7 +123,7 @@ bool isSwitch2POSWarningStateAvailable(int state);
 #endif
 
 #if defined(LIBOPENUI)
-#define IS_INSTANT_TRIM_ALLOWED()     false
+#define IS_INSTANT_TRIM_ALLOWED()     true
```

Comment:
Er, is this right?

Reply:
I think this is a remainder of another attempt to fix something else ;-)